contributed_definitions/NXcontainer.nxdl.xml

```diff
@@ -151,12 +151,14 @@
 			adsorption of the container to be calculated.
 		</doc>
 	</group>
-	<group name="empty_measurement" type="NXentry">
+	<group name="reference_measurement" type="NXentry">
```

Comment:
The revised structure here places a NXentry group (via a link) inside a NXentry group.  That needs to be changed.

replace

	<group name="reference_measurement" type="NXentry">
		<link name="reference_measurement" target="/NXentry"></link>
		<doc>
			A link to a full data collection which contains the actual 
			measured data for this container within the experimental set up 
			(with no sample or inner container(s)). This data set will also 
			include the wavelength/energy, measurement time and intensity for 
			which these data are valid.
		</doc>
	</group>

with

	<link name="reference_measurement" target="/NXentry">
		<doc>
			A link to a full data collection which contains the actual 
			measured data for this container within the experimental set up 
			(with no sample or inner container(s)). This data set will also 
			include the wavelength/energy, measurement time and intensity for 
			which these data are valid.
		</doc>
	</link>